server/src/postgres/mod.rs

```diff
@@ -910,6 +911,7 @@ pub async fn drop_db() -> Result<(), ArrErr> {
     // Drop our tables (in the correct order)
     GenericResource::<flight_plan::Data>::drop_table().await?;
     GenericResource::<vertipad::Data>::drop_table().await?;
+    GenericResource::<itinerary::Data>::drop_table().await?;
```

Comment:
drop order should be the reverse of the create order to prevent possible dependencies.

Reply:
I realised yesterday that I forgot to add vehicles (and fix the vehicle example code), so I'll fix this in my pr today.